integration-tests/tests/test_cases.py

```diff
@@ -86,10 +91,46 @@ def wallet_create_multi_transaction(self):
 
     def setup_method(self):
         super().setup_method()
-        
+
         response = self.wallet_create_multi_transaction()
         try:
-            self.tx_hash = response.json()["result"]["hashes"][str(self.network_id)][0]
+            self.tx_hash = response.json(
+            )["result"]["hashes"][str(self.network_id)][0]
         except (KeyError, json.JSONDecodeError):
             raise Exception(response.content)
 
+
+class SignalTestCase(RpcTestCase):
+
+    received_signals = []
+
+    def _on_message(self, ws, signal):
+        self.received_signals.append(signal)
+
+    def _on_error(self, ws, error):
+        logging.info(f"Error: {error}")
+
+    def _on_close(self, ws, close_status_code, close_msg):
+        logging.info(f"Connection closed: {close_status_code}, {close_msg}")
+
+    def _on_open(self, ws):
+        logging.info("Connection opened")
+
+    def _connect(self):
+        self.url = f"{option.ws_url}/signals"
+
+        ws = websocket.WebSocketApp(self.url,
```

Comment:
Maybe to add this to `try` and log an error, just to know if for example server is down and we cannot connect?